components/brave_rewards/resources/ui/utils.ts

```diff
@@ -125,3 +125,11 @@ export const donationTotal = (report: Rewards.Report) => {
   const tips = new BigNumber(report.tips)
   return new BigNumber(report.donation).plus(tips).dividedBy('1e18').toFixed(1, BigNumber.ROUND_DOWN)
 }
+
+export const constructBackupString = (backupKey: string) => {
```

Comment:
minor: this would be cleaner using template strings. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Done.